pkg/operator/rgw/rgw.go

```diff
@@ -113,7 +113,12 @@ func (c *Cluster) createKeyring(clientset *kubernetes.Clientset, cluster *mon.Cl
 	secrets := map[string]string{
 		keyringName: keyring,
 	}
-	_, err = clientset.Secrets(c.Namespace).Create(&v1.Secret{ObjectMeta: v1.ObjectMeta{Name: appName}, StringData: secrets})
+	secret := &v1.Secret{
```

Comment:
it looks like this is the same thing we're doing with the admin secret in the mon package.  if there's no difference should it be refactored to a single place?